### ISSUE TITLE
Corner case: special "discontinued" spec

### ISSUE THREAD

**Issue by tripu** [w3c/tr-design#107]
eg, [`https://www.w3.org/TR/2016/NOTE-XMLHttpRequest-20161006/`](https://www.w3.org/TR/2016/NOTE-XMLHttpRequest-20161006/). It has no ToC, and the SotD section is the only one.

Per pubrules, we [require a ToC](https://www.w3.org/pubrules/doc/rules/?profile=WG-NOTE#toc), that it [includes `fixup.js`](https://www.w3.org/pubrules/doc/rules/?profile=WG-NOTE#fixupJs), etc. Those things don't make sense in a case like this one, and cause errors.

What do you think, @fantasai, @plehegar, @deniak?

**Comment by tripu** [comment]
I suggest we leave corner cases like this one out of formal rules and checkers: if a spec is discontinued, let's set up a generic "discontinued" page under /TR that catches those cases; possibly with redirections, or additional customisable info.

Otherwise, the rules we have for &ldquo;normal&rdquo; specs won't apply, and we'll have to either relax them in the checkers, make exceptions, or publish manually.

**Comment by deniak** [comment]
Indeed, the TR redesign 2016 introduced the ToC requirement. Since we have a use case where a ToC is completely useless, we could change that requirement into a suggestion instead. Pubrules will not fail if it can't find a ToC.

**Comment by tripu** [comment]
My fear is that if we relax the rules to fit the _lowest common denominator_ we'll end up with only a few rules that won't be very helpful. eg, just because a few weird specs are published without a ToC, we'll now allow every other spec to be published without a ToC? And so on.

**Comment by plehegar** [comment]
One possibility is to have an option to desactivate the ToC requirement, which indeed doesn't make sense here. My guess is that this option would only be applicable in the case of discontinued Note.

**Comment by tripu** [comment]
(Strong candidate for our next cycle; whether that's some time in 2016 or not.)

**Comment by plehegar** [comment]
Most recent discountinued Note: https://www.w3.org/TR/2017/NOTE-tvcontrol-api-20170518/
